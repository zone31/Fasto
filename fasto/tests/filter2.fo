fun bool between(int n) =
  if n < 5 then false
           else (n < 10)

fun int writeInt(int n) = write(n)

fun [int] main() =
  let a = filter(between, {~4, 9, 8, 2, 1, 10, 5, 6, 7, 8, 9, 10}) in //plot everything from range 5 to 9
  map(writeInt, a)

