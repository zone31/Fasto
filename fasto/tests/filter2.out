9 8 5 6 7 8 9
